lua/nvim-treesitter/utils.lua

```diff
@@ -29,14 +31,14 @@ end
 
 function M.get_cache_dir()
   local home = fn.get(fn.environ(), 'HOME')
-  local xdg_cache = fn.get(fn.environ(), 'XDG_CACHE_HOME')
+  local cache_dir = fn.stdpath('data')
```

Comment:
Why not `stdpath('cache')` ?

Reply:
Because then we have to make sure the cache path exists. You gave :+1:  to `stdpath('data')`

Reply:
Okay, sound good then, I wasn't sure in the end, now I am.